migrations/anser.go

```diff
@@ -71,6 +71,7 @@ func (opts Options) Application(env anser.Environment) (*anser.Application, erro
 		// addExecutionToTasksGenerator,
 		oldTestResultsGenerator,
 		testResultsGenerator,
```

Comment:
comment out line 72 and 73 before running

Reply:
done